### PR TITLE
fetch non-public email and trust it.

### PR DESCRIPTION
**Pull request by revir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+
+1. fetch non-public github emails.
+2. automatically set email varified when it's from github.
 
 ## Installation
 
-    npm install nodebb-plugin-sso-github
\ No newline at end of file
+    npm install nodebb-plugin-sso-github2
+
```
Comment (julianlam) [inline README.md:13]: README can be reverted 😄

File: package.json [hunk 2]
```diff
@@ -16,24 +16,19 @@
...
-      "url": "https://github.com/qjmiao/nodebb-plugin-sso-github"
     }
   ],
   "license": "BSD-2-Clause",
   "bugs": {
-    "url": "https://github.com/julianlam/nodebb-plugin-sso-github/issues"
+    "url": "https://github.com/revir/nodebb-plugin-sso-github2/issues"
   },
   "readme": "",
   "readmeFilename": "README.md",
```
Comment (julianlam) [inline package.json:34]: This file can be reverted too, though feel free to keep your name in `contributors` 😃

File: plugin.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-	"id": "nodebb-plugin-sso-github",
-	"name": "NodeBB GitHub SSO",
+	"id": "nodebb-plugin-sso-github2",
+	"name": "NodeBB GitHub SSO 2",
 	"description": "NodeBB Plugin that allows users to login/register via their GitHub account.",
-	"url": "https://github.com/julianlam/nodebb-plugin-sso-github",
+	"url": "https://github.com/revir/nodebb-plugin-sso-github2",
 	"library": "./library.js",
 	"hooks": [
 		{ "hook": "static:app.load", "method": "init" },
```
Comment (julianlam) [inline plugin.json:8]: File can be reverted

**Review by julianlam — CHANGES_REQUESTED** [review]
Looks good, though certain files need to be reverted to their original state. Thanks!

**Comment by revir** [comment]
sorry, I fixed the two problems, and then I wanted to use it on my website, so I changed the readme and package.json, and published to npm, I thought it won't conflict with yours. 
Maybe you can just merge with those previous two changes, and then I will delete my repo, could you?

**Pull request closed without merging**